app/assets/stylesheets/comments.scss

```diff
@@ -413,7 +413,7 @@ a.header-link {
         color: var(--form-placeholder);
       }
       &.embiggened {
-        height: calc(2vw + 115px);
+        height: 125px;
```

Comment:
I can't explain on why ```125px``` fixed this (or why it was ```2w + 115px``` before). I got this value after trial and error.